### PR TITLE
layout file of zh version several links fix

### PR DESCRIPTION
**Pull request by cyjake**
1. the style sheets links
2. change the repo link back to upstream

a minor change of `.gitignore`, do not track vim swap files.

### CODE REVIEW

File: zh/basic/index.html [hunk 1]
```diff
@@ -562,7 +562,7 @@ M hello.rb
     </p>
       
     <h4>
-      git mv
```
Comment (schacon) [inline zh/basic/index.html:565]: this is actually supposed to be 'git mv' - git mv is just a wrapper for the rest of the commands shown below

**Comment by cyjake** [comment]
sorry about that. It was changed to `git rm` when giving a git intro to my colleagues. One pointed out this line looks suspicious and I changed it without thinking any about it.

**Comment by matthewmccullough** [comment]
Thanks for the contribution. The @github/training-team and I have taken the maintenance of this over so we'll be merging pull requests frequently now. We appreciate your efforts!

**Comment by matthewmccullough** [comment]
Merged in via combination with other work in 6906a99

**Pull request closed without merging**